_data/locales/en.yml

```diff
@@ -43,7 +45,16 @@ technologies:
         url: https://github.com/openSUSE/cavil
   container:
     name: Images and Containers
-    description:
+    description: |
```

Comment:
That description doesn't really relate to "Images and Containers" category title

Reply:
Hmm. Does this more relate to Kiwi and Docker in OBS then? I can certainly rewrite it.

Reply:
Indeed